tests/unit_tests/test_msstats.py

```diff
@@ -0,0 +1,152 @@
+"""Test the MSstats script"""
+import logging
+import subprocess
+from pathlib import Path
+
+import pytest
+import pandas as pd
+
+OUTPUTS = [
+    Path("msstats/msstats.input.txt"),
+    Path("msstats/msstats.processed.rda"),
+    Path("msstats.proteins.txt"),
+    Path("msstats.stats.txt"),
+    Path("QCPlot.pdf"),
+]
+
+
+def test_reports(msstats_input, script):
+    """Test without reports"""
+    peptide_file, annot_file, contrast_file = msstats_input
+    args = [
+        script,
+        peptide_file,
+        annot_file,
+        contrast_file,
+        "equalizeMedians",
```

Comment:
I assume equalizeMedians is not the only thing we can do with msstats. What are the other options with msstats? Have we tested anything else?

Reply:
You're absolutely correct. My knowledge might be dated, but MSstats offers 3 options: no normalization, median normalization, and quantile normalization. Each has their place, but generally the more transformations you need to do, the more signal is lost; thus, the general advice is use as much normalization as you need, but no more.

As for testing them, we've not really explored what MSstats can do for us. It is worth keeping in mind that EncyclopeDIA already tries to normalize peptide quants to the total ion current (TIC), which should be proportional to the amount of protein loaded.